ftw/noticeboard/browser/templates/notice.pt

```diff
@@ -4,27 +4,42 @@
       xmlns:i18n="http://xml.zope.org/namespaces/i18n"
       i18n:domain="ftw.noticeboard">
 
+    <head>
+        <div metal:fill-slot="javascript_head_slot" tal:omit-tag="">
+            <script type="text/javascript">
+              (function($){
```

Comment:
Is there a reason why you put that into the template?

Reply:
Yes :-)

1. The JS is only needed for a fraction of sites on this website
2. This way I avoid touching it for plone 5.1 😁